bootstrap.sh

```diff
@@ -296,29 +279,18 @@ install_homebrew() {
   logk
 
   # Update Homebrew.
-  export PATH="$HOMEBREW_PREFIX/bin:$PATH"
-  logn "Updating Homebrew:"
-  [ -z "$QUICK" ] && brew update --quiet
+  export PATH="${HOMEBREW_REPOSITORY}/bin:$PATH"
+  [ -z "$QUICK" ] && {
+    logn "Updating Homebrew:"
+    brew update --quiet
+  }
   logk
 
-  # On Apple M1 machines we need to add this to the profile
-  if [[ "$(uname -m)" == "arm64" ]]; then
-    shell_profile=$(get_brew_profile)
-    eval "$(/opt/homebrew/bin/brew shellenv)"
-    if ! grep -qF "brew shellenv" "${shell_profile}"; then
-      #shellcheck disable=SC2016
-      echo -e 'eval "$(/opt/homebrew/bin/brew shellenv)"' >>${shell_profile}
-    fi
+  shell_rc=$(get_shell_startup_script)
+  if ! grep -qF "brew shellenv" "$shell_rc"; then
+    #shellcheck disable=SC2016
+    echo -e 'eval "$(/opt/homebrew/bin/brew shellenv)"' >> "$shell_rc"
   fi
-
-  # Install Homebrew Bundle, Cask and Services tap.
-  log "Installing Homebrew taps and extensions:"
-  brew bundle --quiet --file=- <<RUBY
-tap "homebrew/cask"
```

Comment:
This stuff isn't necessary at this point and is done as-needed when installing from Brewfiles.